api/graphql/schema.graphql

```diff
@@ -703,6 +707,15 @@ type Query {
     @deprecated(
       reason: "We should avoid non-paginated queries when we know a query will return thousands of values. In this case, we must write an alternative first!"
     )
+  userPublicProfilesPaginated(
+    where: UserPublicProfileFilterInput
+    excludeIds: [ID!] @notIn(key: "id")
+    orderBy: [OrderByClause!] @orderBy
+  ): [UserPublicProfile]!
+    @orderBy(column: "created_at", direction: DESC)
+    @paginate(defaultCount: 10, maxCount: 1000, scopes: ["authorizedToView"])
```

Comment:
Question @tristan-orourke 
Community manager should be able to add people right? Currently they can remove members from a team only, and can only add themselves. I think this is occurring due to the scope being added, but I don't think it should be added as the permission/can and being a limited subset of User should suffice. 

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/713f40e8-7201-4853-99d0-ba6ef29e60a4)

Reply:
Oh this is a good catch. I was actually wondering about the scope. I applied it because it was applied to the original query. Thanks for asking this!

Reply:
@tristan-orourke do you mind taking a look at the rearranging of scopes? Not sure if this was the best way to do it or not 😅 

[fix user basic info query scope](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9296/commits/e650d5fda47399ba9f6edf2ac5b7a3dc0c009464)

Reply:
I also don't know if its the best way or not, but it seems okay 😄

Reply:
lol, okay thanks 👍 🤣